### ISSUE TITLE
Override conditions not working

### ISSUE THREAD

**Issue by RVSkeLe** [BG-Software-LLC/WildStacker#922]
### Minecraft's Version

git-Purpur-1171 (MC: 1.16.5) (Implementing API version 1.16.5-R0.1-SNAPSHOT)

### Plugin's Version

The error started with dev build 37

### Describe the bug

When you try to use WS Override spawner conditions they don't work. I also tried with only WS installed and the issue is still there

### To Reproduce

Try placing spawners high from the ground with override conditions on or place torches near spawners overridden
https://i.imgur.com/sztpSXO.png
https://mclo.gs/I5SuClX

**Comment by OmerBenGera** [comment]
Right click the spawner, check why they cannot spawn
I suspect you have no players nearby which causes this to occur.

**Comment by RVSkeLe** [comment]
> Right click the spawner, check why they cannot spawn I suspect you have no players nearby which causes this to occur.

https://i.imgur.com/DbqR9fs.jpeg, I'm currently in the test server, I was not vanished in the posted,

**Comment by OmerBenGera** [comment]
> > Right click the spawner, check why they cannot spawn I suspect you have no players nearby which causes this to occur.
> 
> https://i.imgur.com/DbqR9fs.jpeg, I'm currently in the test server, I was not vanished in the posted screenshot, as you can see (same config settings as the post before, they just dont want spawn in air/if conditions are not satisfied)

If you run `/ws debug` while looking at the spawner, do you see any messages?

**Comment by RVSkeLe** [comment]
> > > Right click the spawner, check why they cannot spawn I suspect you have no players nearby which causes this to occur.
> > 
> > 
> > https://i.imgur.com/DbqR9fs.jpeg, I'm currently in the test server, I was not vanished in the posted screenshot, as you can see (same config settings as the post before, they just dont want spawn in air/if conditions are not satisfied)
> 
> If you run `/ws debug` while looking at the spawner, do you see any messages in console?

Nothing at all (using latest dev build 62), as soon as spawn conditions are satisfied they spawn, even if they shouldnt have any (disabled with overrides)

**Comment by OmerBenGera** [comment]
Does it happen if you try it on a server with only WildStacker installed?

**Comment by RVSkeLe** [comment]
Yes, same exact thing with only WS and ViaVersion, Backwards, Rewind installed, also 
> when restarting the server the issue seems fixed, but the issue seems to happen as soon as you place new spawners
this still happens, I could send you a video proof, but I doubt that could help you, also note that this started with dev build 37, build 36 is still not affected by this issue

**Comment by OmerBenGera** [comment]
Does it happen on 1.21? As I can't seem to replicate it

**Comment by RVSkeLe** [comment]
> Does it happen on 1.21? As I can't seem to replicate it

In 1.21 using both latest Paper and Purpur builds spawners don't even seem to work, they all display xNum Piggy Spawner, I'll open another issue soon.
Have you tried on 1.16.5?

**Comment by OmerBenGera** [comment]
> > Does it happen on 1.21? As I can't seem to replicate it
> 
> In 1.21 using both latest Paper and Purpur builds spawners don't even seem to work, they all display xNum Piggy Spawner, I'll open another issue soon. Have you tried on 1.16.5?

Spawners are working fine for me on 1.21
Use `/ws give` in order to give yourself spawners, other methods to obtain spawners will not work

**Comment by OmerBenGera** [comment]
Closing this due to lack of response
If the issue still occurs, let me know
